content/get-started/exploring-projects-on-github/contributing-to-a-project.md

```diff
@@ -32,7 +32,7 @@ This tutorial uses [the Spoon-Knife project](https://github.com/octocat/Spoon-Kn
 1. In the top-right corner of the page, click **Fork**.
 
    ![Screenshot of the main page of repository. A button, labeled with a fork icon and "Fork 59.3k," is outlined in dark orange.](/assets/images/help/repository/fork-button.png)
-1. Under "Owner," select the dropdown menu and click an owner for the forked repository.
+1. Under "Owner," select the dropdown menu and click an owner for the forked repository. (If your name is greyed out, it's because you have forked this project in the past; see "Updating an existing fork" below.)
```

Comment:
```suggestion
1. Under "Owner," select the dropdown menu and click an owner for the forked repository.
   >[!NOTE] If your username is grayed out, it's because the fork already exists. Instead, you should bring your existing fork up to date. For more information, see "[AUTOTITLE](/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork#syncing-a-fork-branch-from-the-web-ui)".
```